src/components/pokemon-card/PokemonCard.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import './PokemonCard.css'
+import axios from 'axios'
+
+function PokemonCard({ pokemonName }) {
+    const [pokemon, setPokemon] = useState({});
+
+    useEffect(() => {
+
+        async function fetchSinglePokemon() {
+
+            try {
+                const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`);
+                console.log(response.data);
+                setPokemon(response.data);
+            }
+            catch(e) {
+                console.error(e)
+            }
+        }
+        void fetchSinglePokemon();
+    }, [pokemonName])
+
+    return (
+            <section className="pokemon-card">
+                {Object.keys(pokemon).length > 0 &&
```

Comment:
Dan kan deze dus simpeler
```suggestion
                {pokemon &&
```